components/dashboard/src/prebuilds/list/PrebuildListItem.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+import { TextMuted } from "@podkit/typography/TextMuted";
+import { Text } from "@podkit/typography/Text";
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { TableCell, TableRow } from "@podkit/tables/Table";
+import type { Prebuild } from "@gitpod/public-api/lib/gitpod/v1/prebuild_pb";
+import { useConfiguration } from "../../data/configurations/configuration-queries";
+import { prebuildDisplayProps, prebuildStatusIconName } from "../../projects/Prebuilds";
+import dayjs from "dayjs";
+import { cn } from "@podkit/lib/cn";
+import { shortCommitMessage } from "../../projects/render-utils";
+
+type Props = {
+    prebuild: Prebuild;
+};
+export const RepositoryListItem: FC<Props> = ({ prebuild }) => {
+    const created = dayjs(prebuild.status?.startTime?.toDate()).fromNow();
+    const { data: configuration } = useConfiguration(prebuild.configurationId);
+    const { className: iconColorClass, label } = prebuildDisplayProps(prebuild);
+    const PrebuildStatusIcon = prebuildStatusIconName(prebuild);
+
+    return (
+        <TableRow>
+            <TableCell>
+                <div className="flex flex-col gap-1">
```

Comment:
Make this cell clickable?

Reply:
We could, but I think it's fine to skip for now. We should allow visiting the repo from within the prebuild log UI. There we can make sure the config exists 👍

Reply:
Made it clickable anyway, seems to be quite handy. 

Another thing we could do with it is to apply it as a filter instead, I remember some tables _on the internet_ do this